config/webpack.config.js

```diff
@@ -48,6 +48,13 @@ var config = {
       {
         test: /(\.png|\.gif)$/,
         loader: "url-loader?limit=32767"
+      },
+      {
+        test: /\.scss$/,
+        loader: ExtractTextPlugin.extract(
+          'style', // The backup style loader
```

Comment:
Let's be consistent then, since others are `-loader` please use that for now and open another PR to remove them if you prefer.